src/Page/Options.elm

```diff
@@ -111,126 +111,64 @@ update navKey msg model =
 view : Model -> Html Msg
 view model =
     Search.view { toRoute = Route.Options, categoryName = "options" }
-        "Search NixOS options"
+        [ text "Search more than "
+        , strong [] [ text "10 000 options" ]
+        ]
         model
         viewSuccess
+        viewBuckets
         SearchMsg
 
 
+viewBuckets :
+    Maybe String
+    -> Search.SearchResult ResultItemSource ResultAggregations
+    -> List (Html Msg)
+viewBuckets _ _ =
+    []
+
+
 viewSuccess :
     String
+    -> Bool
     -> Maybe String
-    -> Search.SearchResult ResultItemSource
+    -> List (Search.ResultItem ResultItemSource)
     -> Html Msg
-viewSuccess channel show result =
-    div [ class "search-result" ]
-        [ table [ class "table table-hover" ]
-            [ thead []
-                [ tr []
-                    [ th [] [ text "Option name" ]
-                    ]
-                ]
-            , tbody
-                []
-                (List.concatMap
-                    (viewResultItem channel show)
-                    result.hits.hits
-                )
-            ]
-        ]
+viewSuccess channel showNixOSDetails show hits =
+    ul []
+        (List.concatMap
+            (viewResultItem channel showNixOSDetails show)
+            hits
```

Comment:
I think using `List.map` together with changing API of `viewResultItem` to return `Html msg` would be better here. Concatination of lists here is completely unnecessary

`concatMap` (as well as `concat` and `append`) has a lot of overhead - it's in fact orders of magnitude slower than folding of `(++)` over the list due to implementations details in `elm/core` - we still wait for improvement in core sadly. It's maybe not a big deal as we don't expect to have tens of thousands of items here but considering view function is called many times over the lifetime of application (after every Msg) and that this code will run on many machines I think we can save some wats over time.